src/main/java/com/sponus/sponusbe/member/service/MemberService.java

```diff
@@ -0,0 +1,27 @@
+package com.sponus.sponusbe.member.service;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.sponus.sponusbe.member.dto.MemberJoinRequest;
+import com.sponus.sponusbe.member.entity.Member;
+import com.sponus.sponusbe.member.repository.MemberRepository;
+
+import lombok.RequiredArgsConstructor;
+
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+@Service
+public class MemberService {
+
+	private final MemberRepository memberRepository;
+	private final BCryptPasswordEncoder bCryptPasswordEncoder;
```

Comment:
크게 바뀔일은 없겠지만, 일반적으로는 다형성의 활용을 위해서
BCryptPasswordEncoder를 직접 주입받기 보다는
```suggestion
        private final PasswordEncoder passwordEncoder;
```
와 같이 작성하는 것 같습니다ㅎㅎ